R/PlotEMODResults.R

```diff
@@ -186,20 +185,26 @@ emodplot.by_age_gender <- function(data,
   data['Denominator'] = data[denominator]
     
   # Calculate prevalence, grouping by age bin
-  data.prev <- data %>% group_by(Year, AgeBin_index, AgeBin, Gender, scenario_name, sim.id) %>% 
-    summarize(num = sum(Numerator), denom = sum(Denominator), .groups = 'keep') %>% 
-    mutate(Prevalence = num/denom)
+  data.prev <- data %>% calculate.prevalence.dev(
```

Comment:
I don't see the function `calculate.prevalence.dev` defined anywhere. Is this a typo?

Reply:
Yes this was a typo I'll fix it now